### DISCUSSION TITLE
Remove/hide lifebar elements from K.O. teammates.

### DISCUSSION THREAD

**Discussion by UrsoManso** [ikemen-engine/Ikemen-GO#2163] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

Almost every team based comercial game does this, but for some reason Ikemen doesn't. It would be nice to finally have this feature.

### Describe the solution you'd like

A option(or maybe parameters?) to remove/hide the lifebar elements from teammates who got K.O.

**Comment by rakieldev** [comment, marked as answer]
Duplicated #1142 

We now have several new assertSpecials flags that to disable lifebars
